src/static_data/get_static_data.go

```diff
@@ -0,0 +1,236 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+type Folder struct {
+	FolderName string
+	Urls       []string
+}
+
+func downloadFolder(folder Folder) error {
+	// Create the folder on the file system
+	err := os.MkdirAll(folder.FolderName, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	// Download files to the folder
+	for _, url := range folder.Urls {
+		err := downloadFile(url, folder.FolderName)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func downloadFile(url, folderDir string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("HTTP request failed with status code %d", resp.StatusCode)
+	}
+
+	// Extract the filename from the URL
+	_, filename := filepath.Split(url)
+
+	// Create the file in the specified folder
+	filePath := filepath.Join(folderDir, filename)
+	file, err := os.Create(filePath)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	// Copy the file from the response body to the local file
+	_, err = io.Copy(file, resp.Body)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Downloaded: %s\n", filePath)
+	return nil
+}
+
+func main() {
+	// Create folder instances
+	bgptest1 := Folder{
+		FolderName: "bgptest1",
+		Urls: []string{
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211204.1845.bz2",
```

Comment:
One for the backlog probably, but like comments for some of the other test functions where it'd be good to put data into text files that are loaded in for the test, you could describe the input files for each test in a text file also. Taken a few steps further, a text file per test that describes the the inputs and the expected result.